EIPS/draft-factory-standard.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: <to be assigned>
+title: Standard factory contract format
+description: A standard format for factory contracts
+author: Charles Cooper (@charles-cooper)
+discussions-to: https://github.com/ethereum/EIPs/pull/5202
```

Comment:
This must be a link to a thread on Ethereum Magicians.

Reply:
done in https://github.com/ethereum/EIPs/pull/5202/commits/0f84a168a0331532f0714c0278e7d421a6dcd270